core/consensus/aura/src/lib.rs

```diff
@@ -617,7 +623,7 @@ fn register_aura_inherent_data_provider(
 }
 
 /// Start an import queue for the Aura consensus algorithm.
-pub fn import_queue<B, C, E>(
+pub fn import_queue<B, C, E, P: Pair + Send + Sync + 'static>(
```

Comment:
Send Sync and static are already bounded by Pair, again if it is intentional to make it explicit then ok

Reply:
If pair already binds these, this is unnecessary code.

Reply:
yeah, also any bounds should go in the `where` clause for consistency

Reply:
although I'd prefer the bounds here rather than on `Pair` because they are new in both places.